nixos/modules/programs/sway.nix

```diff
@@ -35,6 +34,17 @@ in {
       Please have a look at the "extraSessionCommands" example for running
       programs natively under Wayland'';
 
+    package = mkOption {
+      type = types.package;
+      default = pkgs.sway-unwrapped;
+      defaultText = "pkgs.sway-unwrapped";
+      example = literalExample "pkgs.sway";
+      description = ''
+        The Sway derivation to use. This can be used to enable
+        the version with a wrapGAppsHook wrapper (pkgs.sway).
+      '';
+    };
+
```

Comment:
Not sure if we need a package option (normally we try to avoid it in cases like this where there are no alternative packages (like e.g. sway-gaps)), but I guess it could become useful if the wrapper has many "use flags" (like `withGAppsWrapper`) and we provide an example.